install/kubernetes/cilium/templates/cilium-configmap.yaml

```
@@ -247,7 +247,7 @@ data:
   enable-bpf-clock-probe: {{ $defaultBpfClockProbe | quote }}
 {{- end }}
 
-{{- if hasKey .Values.bpf "tproxy" }}
+{{- if and (hasKey .Values.bpf "tproxy") (not (kindIs "invalid" .Values.bpf.tproxy)) }}
```

I know I proposed this pattern, but now I wonder: Would the following actually be sufficient?

```suggestion
{{- if (not (kindIs "invalid" .Values.bpf.tproxy)) }}
```

---

ok I will try that, thanks!